File: Problem-Solving-Techniques/Greedy/Python/Absolute_Difference.py
a = [4, 1, 8, 7]
b = [2, 3, 6, 5]

a.sort()
b.sort()

min_diff = 0

for i in range(len(a)):
    min_diff += abs(a[i] - b[i])

print("min absolute value is :", min_diff)

File: Problem-Solving-Techniques/Greedy/C++/Absolute_Difference.cpp
#include <iostream>
#include <algorithm>
#include <cmath>  // for std::abs

int main() {
    int a[] = {4, 1, 8, 7};
    int b[] = {2, 3, 6, 5};
    int n = sizeof(a) / sizeof(a[0]);

    std::sort(a, a + n);
    std::sort(b, b + n);

    int minDiff = 0;

    for (int i = 0; i < n; i++) {
        minDiff += std::abs(a[i] - b[i]);
    }

    std::cout << "min absolute value is : " << minDiff << std::endl;

    return 0;
}
